src/Visualization/Python/PlotDatFiles.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
```

Comment:
I was just thinking that it would be nice to start having some visualization and automation tools in the main repository, so this is great

Is it at all possible to generate a listing and brief description of these scripts in rendered documentation to improve visibility?

Now that we have a few python files, should we put an __init__.py so that python can use the spectre repo tools as a package?

Reply:
I was thinking about ways to expose our Python code better. I think we should expose Python executables (like this one, GenerateXdmf etc) as shell scripts like this:

```sh
#!/bin/bash -e
PYTHONPATH="@CMAKE_BINARY_DIR@/python:$PYTHONPATH" @Python_EXECUTABLE@ -m spectre.@PATH_TO_EXECUTABLE_IN_PACKAGE@
```

This is just so they can be called as `./bin/GenerateXdmf`. The alternative way of calling the scripts, or using the Pybindings, is to `pip install [-e] bin/python`. Then you can just invoke any script as e.g. `python -m spectre.Visualization.GenerateXdmf` or import any bindings as e.g. `import spectre.DataStructures`.

Reply:
A wrapper shell script sounds like a reasonable way to go. I don't think we should force people to pip install. I have no idea what that does, but it makes me worried if someone has multiple repos on their system that things will get very tangled.

Reply:
Re: adding to documentation

I haven't looked into Python+Doxygen interoperability. I would be surprised if this isn't possible, but I don't really have time to figure that out right now, I think.